content/en/continuous_integration/setup_pipelines/buildkite.md

```diff
@@ -10,76 +10,39 @@ further_reading:
-
-[1]: https://app.datadoghq.com/organization-settings/api-keys
-{{< /site-region >}}
+The Datadog integration for [Buildkite][1] works by using [webhooks][2] to send pipeline data to Datadog.
 
-{{< site-region region="eu" >}}
 1. Go to **Settings > Notification Services** in Buildkite and add a new webhook:
-   * **Webhook URL**: `https://webhook-intake.datadoghq.eu/api/v2/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][1].
+   * **Webhook URL**: <code>https://webhook-intake.{{< region-param key="dd_site" >}}/api/v2/webhook/?dd-api-key=<API_KEY></code> where `<API_KEY>` is your [Datadog API key][3].
    * **Events**: Select `job.finished` and `build.finished`.
    * **Pipelines**: Select all pipelines or the subset of pipelines you want to trace.
 
-[1]: https://app.datadoghq.eu/organization-settings/api-keys
-{{< /site-region >}}
-
-{{< site-region region="us3" >}}
-1. Go to **Settings > Notification Services** in Buildkite and add a new webhook:
-  * **Webhook URL**: `https://webhook-intake.us3.datadoghq.com/api/v2/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][1].
-  * **Events**: Select `job.finished` and `build.finished`.
-  * **Pipelines**: Select all pipelines or the subset of pipelines you want to trace.
-
-[1]: https://us3.datadoghq.com/organization-settings/api-keys
-{{< /site-region >}}
-
 2. Click **Add Webhook Notification** to save the new webhook.
 
 ## Visualize pipeline data in Datadog
 
-{{< site-region region="us" >}}
-The [Pipelines][1] and [Pipeline Executions][2] pages populate with data after the pipelines finish.
-
-[1]: https://app.datadoghq.com/ci/pipelines
-[2]: https://app.datadoghq.com/ci/pipeline-executions
-{{< /site-region >}}
-
-{{< site-region region="eu" >}}
-
-The [Pipelines][1] and [Pipeline Executions][2] pages populate with data after the pipelines finish.
-
-[1]: https://app.datadoghq.eu/ci/pipelines
-[2]: https://app.datadoghq.eu/ci/pipeline-executions
-{{< /site-region >}}
+The [Pipelines][4] and [Pipeline Executions][5] pages populate with data after the pipelines finish.
 
-{{< site-region region="us3" >}}
+**Note**: The Pipelines page shows data for only the default branch of each repository.
 
-The [Pipelines][1] and [Pipeline Executions][2] pages populate with data after the pipelines finish.
 
-[1]: https://us3.datadoghq.com/ci/pipelines
-[2]: https://us3.datadoghq.com/ci/pipeline-executions
 {{< /site-region >}}
 
-**Note**: The Pipelines page shows data for only the default branch of each repository.
-
 ## Further reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: https://buildkite.com/docs/apis/webhooks
-[2]: https://buildkite.com
-[3]: /getting_started/tagging/unified_service_tagging
-{{< /site-region >}}
-
-{{< site-region region="us5,gov" >}}
-This feature is not supported for the selected Datadog site ({{< region-param key="dd_site_name" >}}).
-{{< /site-region >}}
+[1]: https://buildkite.com
```

Comment:
Links are not working correctly on this (and other revised CI) pages. I think they have to be inside the site partial?

Reply:
updated, I think the link checker might have move these